src/fragments/start/getting-started/reactnative/integrate.mdx

```diff
@@ -0,0 +1,411 @@
+    textAlign: 'center',
+  },
+  todoContainer: {
+    alignItems: 'center',
+    backgroundColor: '#fff',
+    borderRadius: 2,
+    elevation: 4,
+    flexDirection: 'row',
+    marginHorizontal: 8,
+    marginVertical: 4,
+    padding: 8,
+    shadowOffset: {
+      height: 1,
+      width: 1,
+    },
+    shadowOpacity: 0.3,
+    shadowRadius: 2,
+  },
+  todoHeading: {
+    fontSize: 20,
+    fontWeight: '600',
+  },
+  checkbox: {
+    borderRadius: 2,
+    borderWidth: 2,
+    fontWeight: '700',
+    height: 20,
+    marginLeft: 'auto',
+    textAlign: 'center',
+    width: 20,
+  },
+  completedCheckbox: {
+    backgroundColor: '#000',
+    color: '#fff',
+  },
+  buttonText: {
+    color: '#fff',
+    fontWeight: '600',
+    padding: 16,
+  },
+  buttonContainer: {
+    alignSelf: 'center',
+    backgroundColor: '#4696ec',
+    borderRadius: 99,
+    paddingHorizontal: 8,
+  },
+  floatingButton: {
+    position: 'absolute',
+    bottom: 44,
+    elevation: 6,
+    shadowOffset: {
+      height: 4,
+      width: 1,
+    },
+    shadowOpacity: 0.3,
+    shadowRadius: 4,
+  },
+  modalContainer: {
+    backgroundColor: 'rgba(0,0,0,0.5)',
+    flex: 1,
+    justifyContent: 'center',
+    padding: 16,
+  },
+  modalInnerContainer: {
+    backgroundColor: '#fff',
+    borderRadius: 16,
+    justifyContent: 'center',
+    padding: 16,
+  },
+  modalInput: {
+    borderBottomWidth: 1,
+    marginBottom: 16,
+    padding: 8,
+  },
+  modalDismissButton: {
+    marginLeft: 'auto',
+  },
+  modalDismissText: {
+    fontSize: 20,
+    fontWeight: '700',
+  },
+});
+
+export default Home;
+```
+
+Go ahead and run your code now and you should see an app with empty todolist and a floating action button but not much else.
+
+<BlockSwitcher>
+<Block name="Expo">
+
+```bash
+expo start
+```
+
+</Block>
+<Block name="React Native CLI">
+
+To run on iOS, execute the following command:
+```bash
+npx react-native run-ios
+```
+
+To run on Android, execute the following command:
+```bash
+npx react-native run-android
+```
+
+</Block>
+</BlockSwitcher>
+
+Let’s next manipulate the data and add more functionality to our app.
+
+## Manipulating data
+
+### Creating a Todo
+The *Add todo* button opens up a Modal to add todos. But, right now, the form does nothing when the *Save* button is pressed. Let’s fix that by having it save a **Todo** to DataStore.
+
+Update the `addTodo()` function in the **AddTodoModal** component.
+
+```js
+  async function addTodo() {
+    await DataStore.save(new Todo({ name, description, isComplete: false }));
+    setModalVisible(false);
+    setName('');
+    setDescription('');
+  }
+```
+
+If you try to add todos using the form now, it should successfully close the form when pressing the *Save* button. But our Todo list is still empty even if you restart the app! After initializing our todos as an empty list, we aren't currently updating it again. We will remedy that in the next step.
+
+### Querying for Todos
```

Comment:
I think this section and the next should be combined and updated to use the new `observeQuery` API. See Flutter: https://docs.amplify.aws/start/getting-started/integrate/q/integration/flutter/#querying-for-todos-and-observing-updates-in-real-time

Reply:
I have update the tutorial to use `observerQuery` instead of `observe`. The code is much simpler now.